cmd/client/main.go

```diff
@@ -1,9 +1,27 @@
 package main
 
 import (
+	"flag"
+
 	"github.com/ASeegull/SmartFridge/staticServer"
+	"github.com/ASeegull/SmartFridge/staticServer/config"
+	"github.com/davecheney/errors"
+	log "github.com/sirupsen/logrus"
 )
 
 func main() {
-	staticServer.Run()
+	cfgPath := flag.String(
+		"config",
+		"staticServer/config/files/clientConfig.yaml",
+		"Location of config File",
+	)
+
+	log.Info(*cfgPath)
+	flag.Parse()
+	cfg, err := config.GetSettings(*cfgPath)
+	if err != nil {
+		log.Fatal(errors.Annotate(err, "Server couldn't start"))
```

Comment:
Maybe errors.Wrap() from standard library can be used here?
Standart libraries are always preferable